test/README.md

```diff
@@ -46,6 +46,7 @@ The end-to-end tests take several parameters, which they expect to find in a fil
 | `tempFilterFuncImage` | * | Azure temperature filter function to be used. Required when running the test 'TempFilterFunc', ignored otherwise.|
 | `tempFilterImage` | * | Docker image to pull/use for the temperature filter module. Required when running the test 'TempFilter', ignored otherwise.|
 | `tempSensorImage` || Docker image to pull/use for the temperature sensor module (see the test 'TempSensor'). If not given, `mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0` is used.|
+| `numberLoggerImage` || Docker image to pull/use for the direct method tests. Used to generate predictable logs. |
```

Comment:
Nit: "Docker image to pull/use for the **Edge agent** direct method tests..." (since there are other direct method tests which don't use this module)